src/app/kbv/controllers/self-assessment-question.js

```diff
@@ -0,0 +1,52 @@
+const debug = require("debug")("load-question");
+const BaseController = require("hmpo-form-wizard").Controller;
+const { submitAnswer, getNextQuestion } = require("../service");
+const taxYearToRange = require("../../../utils/tax-year-to-range");
+
+class SelfAssessmentQuestionController extends BaseController {
+  locals(req, res, callback) {
+    super.locals(req, res, (err, locals) => {
+      if (err) {
+        return callback(err, locals);
+      }
+
+      locals.info = taxYearToRange(req.session.question.info.currentTaxYear);
+
+      callback(null, locals);
+    });
+  }
+
+  async saveValues(req, res, callback) {
+    await super.saveValues(req, res, async (err) => {
+      if (err) {
+        return callback(err);
+      }
+
+      try {
+        const userInput = JSON.stringify({
+          statePension: req.body.statePension,
```

Comment:
The way we serialise this answer for the backend is TBC, but we should cater for both sets of fields (so we don't forget!) - so this will need to be something like
```
statePension: req.body.statePension || req.body.statePensionShort,
...
```